source/sass/blocks/extra-product-catalog.scss

```
@@ -55,21 +73,75 @@
   color: v.$white-color;
   background-image:
     linear-gradient(to top, rgba(104, 183, 56, 0.85), rgba(104, 183, 56, 0.85)),
-    url("../img/cat-present.jpg");
+    url("../img/cat-present/cat-present-mobile.jpg");
+  background-size: cover;
   background-repeat: no-repeat;
-  position: relative;
-  display: flex;
-  justify-content: center;
-  align-items: end;
-  padding: 59px 61px 43px 58px;
+  background-color: v.$brand-color-500;
+  padding: 187px 60px 43px 60px;
   text-align: center;
+  position: relative;
+  margin: 35px 0;
+
+  @media (min-resolution: 2dppx) {
+    background-image:
+      linear-gradient(to top, rgba(104, 183, 56, 0.85), rgba(104, 183, 56, 0.85)),
```

Можно еще переменные использовать, препроцессор умеет их раскрывать при компиляции:
```suggestion
      linear-gradient(to top, rgba($some-color, 0.85), rgba($some-color, 0.85)),
```

Это не обязательно применять, просто к сведению =)
Иногда бывает очень удобно